.changeset/blue-dots-battle.md

```diff
@@ -0,0 +1,27 @@
+---
+"@primer/react-brand": minor
+---
+
+Added `OrderedList` and `UnorderedList`
+
+Example Usage:
+
+```
+<UnorderedList variant="checked">
+    <li>Automatic security and version updates</li>
+    <li>GitHub Security Advisories</li>
+    <li>Code and secret scanning</li>
+    <li>Dependency review</li>
+    <li>Automated authentication and identity management</li>
+</UnorderedList>
+```
+
+```
+<OrderedList>
+    <li>Automatic security and version updates</li>
+    <li>GitHub Security Advisories</li>
+    <li>Code and secret scanning</li>
+    <li>Dependency review</li>
+    <li>Automated authentication and identity management</li>
+</OrderedList>
+```
```

Comment:
Can you link to the docs page from here too please? 

Good examples in the previous changesets for new components.

Reply:
In next push